### DISCUSSION TITLE
How to make it send `stopCharging` actions by itself

### DISCUSSION THREAD

**Discussion by zhaolin-yang** [openTCS/opentcs#114] [Q&A]
![config](https://github.com/openTCS/opentcs/assets/65264428/de257448-41fb-4bd9-b59e-54d9d937db24)
    I configured the properties shown in the figure on location. This location is a charging station, which can start charging when the vehicle arrives at the location, but when the battery level meets the condition, there is no way to stop charging.
    The actions in a order issued via VDA5050 are shown below:
![VDA](https://github.com/openTCS/opentcs/assets/65264428/6d3769c7-3692-4d6f-afff-bf68d9ab0542)
    What can I do to stop charging properly?

**Comment by hitmangithub** [comment]
What's you company?In China?

**Comment by swltr** [comment]
@zhaolin-yang Usually, I'd say it would make sense for the vehicle to stop charging automatically once it gets a new order, not when the battery level has reached a certain level.
